lib/features/guide_view/widgets/about_us_section.dart

```diff
@@ -0,0 +1,36 @@
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../theme/app_theme.dart";
+import "../../../utils/context_extensions.dart";
+import "../../../widgets/wide_tile_card.dart";
+import "../../navigator/utils/navigation_commands.dart";
+
+class GuideAboutUsSection extends ConsumerWidget {
+  const GuideAboutUsSection({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Theme(
+      data: Theme.of(context).getAlTitlesOrange(context),
+      child: PhotoTrailingWideTileCard(
+        title: context.localize.guide_about_us,
+        subtitle: context.localize.guide_meet_creators,
+        photoUrl: "",
+        onTap: ref.navigateAboutUs,
+      ),
+    );
+  }
+}
+
+extension _ChangeTitleToOrangeTitleX on ThemeData {
```

Comment:
### TODO:
Refactor this to use global util: https://github.com/Solvro/mobile-topwr/blob/06d6f505b63a161004ba3b88885029db724754c7/lib/theme/utils.dart
once everyhting is merged